test-site/jambo-yext-sites.json

```diff
@@ -0,0 +1,18 @@
+{
+  "dirs": {
+    "themes": "../../",
+    "config": "config",
+    "output": "public",
+    "pages": "pages",
+    "partials": [
+      "script/on-ready.js",
+      "cards",
+      "directanswercards"
+    ],
+    "preservedFiles": [
+      "public/iframe_test.html",
+      "public/overlay.html"
+    ]
+  },
+  "defaultTheme": "repo"
```

Comment:
I remember running pwd before in SGS and I think it rename the top level folder to `/repo-vol`?

Reply:
For full builds, it renames the top level folder to `repo`. For live preview, it looks like it uses 'repo-vol'. I could try making another file which refers to `repo-vol` for live preview